sprint9/src/components/Nav.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import { useContext } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import style from "../styles/Nav.module.css";
+import pcLogo from "../../public/assets/pndamarket_logo.png";
+import mobileLogo from "../../public/assets/m_pandamarket_logo.png";
+import { AuthContext } from "@/src/context/AuthProvider";
+import profileImg from "@/public/assets/img_profile.png";
+import { usePathname } from "next/navigation";
+
+export default function Nav() {
+  const { user } = useContext(AuthContext);
+  const currentPathname = usePathname();
+
+  const profileDefaultImg = user?.image ? user.image : profileImg;
```

Comment:
@annylee0723 아래처럼 쓸 . 수있어요
```suggestion
  const profileDefaultImg = user?.image ?? profileImg;
```

```suggestion
  const profileDefaultImg = user?.image || profileImg;
```